src/ResourceManager/Sql/Commands.Sql.Test/Commands.Sql.Test.csproj

```diff
@@ -78,8 +78,9 @@
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Azure.Management.Sql, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <!--<HintPath>..\..\..\packages\Microsoft.Azure.Management.Sql.1.10.0-preview\lib\net452\Microsoft.Azure.Management.Sql.dll</HintPath>-->
+	  <HintPath>F:\GitHub\azure-sdk-for-net\src\SDKs\SqlManagement\Management.Sql\bin\Debug\net452\Microsoft.Azure.Management.Sql.dll</HintPath>
```

Comment:
@payiAzure please revert the change made here -- if you want to use functionality from a newly generated Sql SDK, you should do the following:

- create a new _signed_ SDK using the `NetCore-SDK-Sign` job on our Jenkins server (use the fork and branch containing your SDK changes as the parameters for the job)
    - make sure that the version of the SDK is bumped to `1.14.0-preview`
- place the generated SDK in the `tools/LocalFeed` folder in the PowerShell repository
- bump the version of the SDK used in PowerShell to `1.14.0-preview`